tests/dummy/app/pods/docs/quickstart/template.md

```diff
@@ -117,7 +117,7 @@ Since most guides contain written prose, the majority of your guides pages will
 The generator should have created
 
   - the **markdown file** in the `tests/dummy/app/templates/docs` directory
-  - the **nav item entry** in `tests/dummy/app/templates/docs.md`
+  - the **nav item entry** in `tests/dummy/app/templates/docs.hbs`
```

Comment:
Does this have to be `hbs`? Most files support `md` I believe.

Reply:
This is the actual route, not the doc file. That said, this isn't about documenting what we're asking the user to do, this is about documenting what the generator actually *does*. Having literally run the generator right before I made this PR, I can definitively confirm that the file we're dealing with here is a .hbs file, as the Quickstart instructs you to create here: https://ember-learn.github.io/ember-cli-addon-docs/docs/quickstart#6-create-your-docs-skeleton